test/e2e/memcached_test.go

```diff
@@ -574,3 +581,88 @@ func MemcachedClusterTest(t *testing.T) {
 		t.Fatalf("In-cluster test failed: %v\nCommand Output:\n%s", err, string(cmdOut))
 	}
 }
+
+func memcachedMetricsTest(t *testing.T, f *framework.Framework, ctx *framework.TestCtx) error {
+	operatorName := "memcached-operator"
```

Comment:
We should refactor `operatorName` to a package-level `const`, since its used in several places above.